about.html

```diff
@@ -295,7 +318,7 @@ <h5 class="text-center mt-50"> More <span class="color-blue5"> 15,000 </span> Co
                 <div class="content section-padding">
                     <div class="section-head text-center mb-70 style-5">
                         <h2 class="mb-20"> News & <span> Insights </span> </h2>
-                        <p>More than x companies trust and choose AICTA</p>
+                        <p>More than 1000 companies trust and choose AICTA</p>
```

Comment:
This is a brand new company, so we won't be able to claim that 1000 companies trust us. I feel that it's better to remove this line for now.

Also look into the career form section above (lines 255 to 313) and look at modifying/removing that section. _I'm not able to highlight them, but I've left a screenshot below._

![image](https://user-images.githubusercontent.com/8443215/229262099-5b00886e-9a59-45ea-be9a-7170be92eb6a.png)

Reply:
Thanks for doing that Michael. Just need to go to Line 321 as well. Can you remove the subheadline here since it says 1000 companies trust AICTA?